src/apps/adgangsplatformen/user-token.dev.tsx

```diff
@@ -0,0 +1,12 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import UserToken from "./user-token";
+
+export default {
+  title: "Set User Token",
```

Comment:
I think this should be organized under: "SB Utilities / ..."
Like [here](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/release/2/src/apps/adgangsplatformen/library-token.dev.tsx#L6)

Reply:
It did not exists when I made this... but will move it

Reply:
Great be aware that it only implies a change in the title. No files need to be moved ;)